### PR TITLE
Add kucoin.com

### PR DESCRIPTION
**Pull request by closehandle**

### CODE REVIEW

File: data/kucoin [hunk 1]
```diff
@@ -0,0 +1,3 @@
+kucoin.com
+kucoin.plus
+staticimg.com
```
Comment (rootmelo92118) [inline data/kucoin:3]: Please provide evidence of this domain belonging to this category.

**Review by rootmelo92118 — CHANGES_REQUESTED** [review]

File: data/kucoin (hunk 1 above)
Comment (rootmelo92118) [reply data/kucoin:3]: No. I mean the `staticimg.com`

Comment (closehandle) [reply data/kucoin:3]: KuCoin hosts static resources on it. I searched through all the rules and didn't find any other references to `staticimg.com`, so I added it in

Comment (rootmelo92118) [reply data/kucoin:3]: > 
> 
> KuCoin hosts static resources on it. I searched through all the rules and didn't find any other references to `staticimg.com`, so I added it in

No, I mean... Are you sure this domain is only use on this website?

**Comment by rootmelo92118** [comment]
According the searching result in google, I am very sure this doamin is not just use on this website.
image

**Pull request merged**